bot/exts/info/code_snippets.py

```diff
@@ -224,13 +224,18 @@ def __init__(self, bot: Bot):
     async def on_message(self, message: Message) -> None:
         """Checks if the message has a snippet link, removes the embed, then sends the snippet contents."""
         if not message.author.bot:
-            message_to_send = ''
+            all_snippets = []
 
             for pattern, handler in self.pattern_handlers:
                 for match in pattern.finditer(message.content):
-                    message_to_send += await handler(**match.groupdict())
+                    snippet = await handler(**match.groupdict())
+                    all_snippets.append((match.start(), snippet))
 
-            if 0 < len(message_to_send) <= 2000 and message_to_send.count('\n') <= 15:
+            # Sorts the list of snippets by their match index and joins them into
+            # a single message
```

Comment:
```suggestion
            # Sorts the list of snippets by their match index and joins them into a single message
```
If I'm not mistaken, this should be fine given the config line below for a max line length of 120.

https://github.com/python-discord/bot/blob/main/tox.ini#L2